src/react-components/info-dialog.js

```diff
@@ -183,11 +185,45 @@ class InfoDialog extends Component {
           </span>
         );
         break;
+      case InfoDialog.dialogTypes.help:
+        dialogTitle = "How to Play";
+        dialogBody = (
+          <div className="info-dialog__help">
+            When in a room, other avatars can see and hear you.
+            <p />
+            Use your controller&apos;s action button to teleport from place to place. If it has a trigger, use it to
+            pick up objects.
+            <p style={{ textAlign: "center" }}>
+              In VR, <b>look up</b> to find your menu:
+              <img
+                className="info-dialog__help__hud"
+                src="../assets/images/help-hud.png"
+                srcSet="../assets/images/help-hud@2x.png 2x"
+              />
+            </p>
+            <p>
+              The <b>Mic Toggle</b> mutes your mic.
+            </p>
+            <p>
+              The <b>Pause/Resume Toggle</b> pauses all other avatars. You can then block them from having further
+              interactions with you.
+            </p>
+            <p>
+              The <b>Bubble Toggle</b> hides avatars that enter your personal space.
+            </p>
+          </div>
```

Comment:
Help content is too tall for iPhone 5/SE and Galaxy S5

Reply:
Also worth mentioning that we can/should get a whole lot smarter about help text. For example, we should have specific copy for each device. So the next pass can probably address this problem since a lot of the copy on an S5 is irrelevant and can be removed since it's not VR capable, etc.